e2e/ansible/playbooks/resiliency/test-single-replica-failure/single-replica-sc.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: single-replica-sc
+  annotations:
+    cas.openebs.io/create-volume-template: jiva-volume-create-default-0.7.0
+    cas.openebs.io/delete-volume-template: jiva-volume-delete-default-0.7.0
+    cas.openebs.io/read-volume-template: jiva-volume-read-default-0.7.0
+    cas.openebs.io/config: |
+      - name: ControllerImage
+        value: openebs/jiva:0.6.0
+      - name: ReplicaImage
+        value: openebs/jiva:0.6.0
+      - name: VolumeMonitorImage
+        value: openebs/m-exporter:ci
+      - name: ReplicaCount
+        value: "1"
+      - name: StoragePool
+        value: default
```

Comment:
@dargasudarshan , would it be helpful to use template here with image passed to test

Reply:
Done sir